lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -46,6 +47,7 @@ registerType("normal", "n", 0,
 E2Lib.registerConstant("PI", pi)
 E2Lib.registerConstant("E", exp(1))
 E2Lib.registerConstant("PHI", (1+sqrt(5))/2)
+E2Lib.registerConstant("HUGE", huge)
```

Comment:
Why call it "HUGE" and not "INFINITY" or something? ("INF" was suggested in #1143)